ui/drivers/qt/ui_qt_window.cpp

```diff
@@ -277,13 +247,14 @@ MainWindow::MainWindow(QWidget *parent) :
    QMainWindow(parent)
    ,m_loadCoreWindow(new LoadCoreWindow(this))
    ,m_timer(new QTimer(this))
+   ,m_thumbnailTimer(new QTimer(this))
```

Comment:
This needs to go at the bottom of the init list, as in my diff here that I provided earlier:
```c
diff --git a/ui/drivers/qt/ui_qt_window.cpp b/ui/drivers/qt/ui_qt_window.cpp
index ccec679b9d..579a7142b6 100644
--- a/ui/drivers/qt/ui_qt_window.cpp
+++ b/ui/drivers/qt/ui_qt_window.cpp
@@ -247,7 +247,6 @@ MainWindow::MainWindow(QWidget *parent) :
    QMainWindow(parent)
    ,m_loadCoreWindow(new LoadCoreWindow(this))
    ,m_timer(new QTimer(this))
-   ,m_thumbnailTimer(new QTimer(this))
    ,m_currentCore()
    ,m_currentCoreVersion()
    ,m_statusLabel(new QLabel(this))
@@ -322,6 +321,7 @@ MainWindow::MainWindow(QWidget *parent) :
    ,m_failedThumbnails(0)
    ,m_playlistThumbnailDownloadWasCanceled(false)
    ,m_pendingDirScrollPath()
+   ,m_thumbnailTimer(new QTimer(this))
 {
    settings_t *settings = config_get_ptr();
    QDir playlistDir(settings->paths.directory_playlist);
```

Reply:
Hey @CozmoP , if you can address these last few issues that @bparker06 pointed out, I think we can start fast-tracking this for merging.

Reply:
Hey @CozmoP , another request to address these issues by @bparker06 , then we can finally merge this.

Reply:
Hi @twinaphex , I believe I addressed these issues, but see my last comment.